static/panes/cfg-view.js

```diff
@@ -102,25 +102,29 @@ function Cfg(hub, container, state) {
     this._editorid = state.editorid;
     this._binaryFilter = false;
 
-    this.functionPicker = $(this.domRoot).find('.function-picker').selectize({
+    var self = this;
+    var pickerEl = this.domRoot[0].querySelector('.function-picker');
```

Comment:
FWIW I have no strong opinion on this, raw dom vs $ stuff. I'd like to phase out jquery, but...it's a great way of "just working" on janky browsers. I _do_ like the fact you're doing the `[0]` stuff, much nicer. thanks.

Reply:
Speaking of janky browsers, I should probably note that tom-select does not officially work on IE 11. If you wanted to use this and support IE 11, tom-select would need to be built with different browserslist.

Reply:
Got it. We'll have to see on that front. From Google stats (those that allow us to track) we have had one internet explorer in the last month...trawling the server logs I see 12,168 hits with 'rv:11.' in the user agent which seems to be the right magic to find, so it's not "no users". (out of 22,579,501 hits in the logs, so...not many)